internal/provider/cluster.go

```diff
@@ -135,11 +135,7 @@ func ResourceCluster() *schema.Resource {
 				Description: "Whether to enable CNAME for seed nodes",
 				Optional:    true,
 				Type:        schema.TypeBool,
-				// NOTE(rjeczalik): ForceNew is commented out here, otherwise
-				// internal provider validate fails due to all the attrs
-				// being ForceNew; Scylla Cloud API does not allow for
-				// updating existing clusters, thus update the implementation
-				// always returns a non-nil error.
+				// TODO: Make ForceNew to work
```

Comment:
I guess this problem cannot be fixed in the provider - scylla cloud api needs to be updated to allow for update operations (rescale etc.)

Till then I think it's best to keep the original long explanation